foo

```diff
@@ -0,0 +1 @@
+foo
```

Comment:
forgot to remove?:p

Reply:
Thanks for the catch! Created this file when I was testing something else